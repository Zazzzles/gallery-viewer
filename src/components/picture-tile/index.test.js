import React from 'react';
import renderer from 'react-test-renderer';

import PictureTile from './index';
import NavigationContext from '../../context/navigation-context';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <NavigationContext.Provider value={{ xIndex: 0 }}>
        <PictureTile
          urls={{
            regular:
              'https://hips.hearstapps.com/countryliving.cdnds.net/17/47/1511194376-cavachon-puppy-christmas.jpg',
          }}
        />
      </NavigationContext.Provider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
