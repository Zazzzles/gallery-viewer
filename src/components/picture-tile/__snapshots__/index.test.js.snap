// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Container"
>
  <img
    alt="Content"
    className="Image"
    loading="eager"
    onLoad={[Function]}
    src="https://hips.hearstapps.com/countryliving.cdnds.net/17/47/1511194376-cavachon-puppy-christmas.jpg"
  />
</div>
`;
